.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -12,88 +13,81 @@ on:
         type: boolean
         description: 'Debug mode (run without publishing).'
         default: false
+
 jobs:
-  release:
-    name: Prepare Deployment
+  pre-run:
+    name: Pre-run
     runs-on: ubuntu-latest
     outputs:
-      matrix: ${{ steps.set-matrix.outputs.matrix }}
+      matrix: ${{ steps.set-matrix.outputs.json }}
```

Comment:
What does this change do?

Reply:
It simplifies the matrix for the plugins list. Now we just provide the array of plugins in the matrix and all other data is fetched from build manifest.